benchmarks/package.json

```
@@ -5,7 +5,8 @@
     "bench": "vitest bench --run"
   },
   "dependencies": {
-    "nodegit": "link:./nodegit"
+    "nodegit": "link:./nodegit",
+    "simple-git": "link:./simple-git"
```

Since, `@napi-rs/simple-git` not uses node-gyp and includes prebuilt across all platforms, how about installing it directly with the yarn workspace module?
```suggestion
    "@napi-rs/simple-git": "0.1.19"
```

---

Oh, I thought it's conversion of benchmark folder.
I'll change it when I possible to work.

---

Since nodegit does not have a prebuilt for workspaces Node.js version, so installation with the yarn workspace module may be failed depending on the users environment. Therefore, we are safely separating the package and installing it manually. 😭 
For detailed instructions, please refer to [benchmarks/README.md](https://github.com/toss/es-git/tree/main/benchmarks).

---

@seokju-na I updated commit for this review